### DISCUSSION TITLE
Manage API Exposed Schemas with Supabase CLI or Management API

### DISCUSSION THREAD

**Discussion by evanrs** [supabase/supabase#19241] [Feature Requests]
The settings found under [supabase.com/dashboard/project/:project-id/settings/api#project-postgres-config](https://supabase.com/dashboard/project/:project-id/settings/api#project-postgres-config) have no documented way of being managed programmatically.

Exposing this in the Supabase CLI or just the Management API would assist in automating the deployment of new projects.

Requiring user intervention to expose additional schema limits what can be built on Supabase. Module like extensions to Supabase would prefer not to pollute the `public` schema and instead work within their own namespace. Adding this feature will improve the Supabase ecosystem.

**Comment by DanMossa** [comment]
Has there been any updates to allow this?
